### ISSUE TITLE
Feature Requests

### ISSUE THREAD

**Issue by MacroMachines** [szymonkaliski/SDF-UI#8]
There are some very cool additional SDF combine ops in the HG_SDF source such as the fOpUnionChamfer and fOpDifferenceStairs etc..

I might suggest for tidiness and ease of access, creating an additional level of hierarchy to the create context menu with something like:
primitive
domain
combine
modify
math
i/o (system)

**Comment by MacroMachines** [comment]
http://mercury.sexy/hg_sdf/

**Comment by szymonkaliski** [comment]
Yeah, I've stolen simple ones from there, if you want to add more the interface should be pretty straight-forward, look here for sample: https://github.com/szymonkaliski/SDF-UI/blob/master/src/engine/nodes/op-difference-round.js

+1 on the additional level, but as I mentioned this was made just for fun and I don't think I'll get back to this project soon, have other things in pipeline rn; happy to merge PRs though!

**Comment by MacroMachines** [comment]
Yeah, Im down to fork it and add / continue.  Is this mostly JavaScript?  Any suggestions for IDE?  I have eclipse but not a huge fan of it thus far.  I do a lot of coding in sublime text 3.

**Comment by szymonkaliski** [comment]
Yes, it's all JS with small bits of inline GLSL. I do VIM for everything, but that needs some getting used to... I think Sublime should be good for JS, a lot of folks are also using Atom I believe.
